roles/openshift_logging/vars/main.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
 es_node_quorum: "{{openshift_logging_es_cluster_size|int/2 + 1}}"
+es_min_masters_default: "{{ (openshift_logging_es_cluster_size | int / 2 | round(0,'floor') + 1) | int }}"
```

Comment:
http://jinja.pocoo.org/docs/2.9/templates/#round the first argument is actually the data passed in via the pipe